shared-libs/validation/src/validation.js

```diff
@@ -0,0 +1,316 @@
+const _ = require('lodash/core');
```

Comment:
I moved this file but GH still reports full edits :( 
I tried to keep changes to a minimum, but now that it's 100% changed, I'm wondering if we should prettify it even more.